### PR TITLE
allow disabling TLS for kubelet

### PR DESCRIPTION
**Pull request by jaxxstorm**

### CODE REVIEW

File: charts/vantage-kubernetes-agent/values.yaml [hunk 1]
```diff
@@ -32,6 +32,9 @@ agent:
   # Optional. Comma separated list of allowed annotations to be sent to Vantage. Max 10 annotations. Note that values are truncated after 100 chars.
   allowedAnnotations: ""
 
+  # Optional. Some cluster providers have self signed certificates on the kubelet. If you are seeing TLS errors, you can disable TLS verification here.
+  disableKubeletTLSverify: "false"
```
Comment (macb) [inline charts/vantage-kubernetes-agent/values.yaml:36]: Given the changes, I think this should be `disableKubeTLSverify` and the comment updated to reflect it disables tls verify for all kubernetes related requests (api server and node metrics server).

**Comment by macb** [comment]
Let's also bump the [chart (increment) and app versions](https://github.com/vantage-sh/helm-charts/blob/main/charts/vantage-kubernetes-agent/Chart.yaml#L5-L6) for the new agent that supports this.

File: charts/vantage-kubernetes-agent/values.yaml (hunk 1 above)
Comment (jaxxstorm) [reply charts/vantage-kubernetes-agent/values.yaml]: Good call out, fixed!

**Comment by jaxxstorm** [comment]
> Let's also bump the [chart (increment) and app versions](https://github.com/vantage-sh/helm-charts/blob/main/charts/vantage-kubernetes-agent/Chart.yaml#L5-L6) for the new agent that supports this.

Will push to this branch once the changes to the container are shipped!

**Review by macb — APPROVED** [review]
Image version `1.0.18` is now available.

**Pull request merged**